### DISCUSSION TITLE
Should be useful to change the channel frequency

### DISCUSSION THREAD

**Discussion by piercalderan** [sandeepmistry/arduino-LoRa#479] [General]
I think it would be useful to be able to change the txmode and rxmode with two different frequencies. For example with txmode set the frequency with LoRa.setFrequency (868300000UL) and for rxmode LoRa.setFrequency (868100000UL).

**Comment by IoTThinks** [comment]
You can change the frequency before sending and set the frequency before returning to receive mode anytime.

**Reply by piercalderan** [reply]
Thank you very much. So I can write, e.g., the method LoRa.setFrequency (868100000UL) to change the frequency. In this case I can use, for example one channel for tx and one channel for rx, not simultaneously, of course.
